File: news/src/main/java/material/com/news/model/NewsEntity.java
package material.com.news.model;

import java.util.List;

/**
 * Created by zjl on 2017/3/28.
 */

public class NewsEntity {
    private boolean error;
    private List<NewsItem> results;

    public boolean isError() {
        return error;
    }

    public void setError(boolean error) {
        this.error = error;
    }

    public List<NewsItem> getResults() {
        return results;
    }

    public void setResults(List<NewsItem> results) {
        this.results = results;
    }

    @Override
    public String toString() {
        return "NewsEntity{" +
                "error=" + error +
                ", results=" + results +
                '}';
    }
}

File: news/src/main/java/material/com/news/model/NewsEntity.kt
package material.com.news.model

/**
 * Created by zjl on 2017/3/28.
 */

class NewsEntity {
    var isError: Boolean = false
    var results: List<NewsItem>? = null

    override fun toString(): String {
        return "NewsEntity{" +
                "error=" + isError +
                ", results=" + results +
                '}'
    }
}
